mesonbuild/coredata.py

```diff
@@ -604,6 +604,28 @@ def __init__(self, options: SharedCMDOptions, scratch_dir: str, meson_command: T
         self.builtin_options_libdir_cross_fixup()
         self.init_builtins('')
 
+        self._is_native_clone = False
+
+    def copy_to_native(self) -> CoreData:
+        other = CoreData.__new__(CoreData)
+        for k, v in self.__dict__.items():
+            other.__dict__[k] = v
+
+        other.cross_files = []
+
+        other.compilers = PerMachine(OrderedDict(), OrderedDict())
+        other.compilers.build = self.compilers.build
+        other.compilers.host = self.compilers.build
```

Comment:
This shoudl be able to be represented as

other.compilers = PerMachine(self.compilers.build, self.compilers.build)

Reply:
Oops, yes, thanks! Fixed.